src/screens/Followings/elements/RenderList/index.js

```diff
@@ -60,12 +60,12 @@ const styles = StyleSheet.create({
     fontFamily: fonts.inter[500],
     fontWeight: 'bold',
     fontSize: 14,
-    color: colors.black,
+    color: COLORS.black,
     lineHeight: 16.94
   },
   textProfileFullName: {
     fontSize: 12,
-    color: colors.gray,
+    color: COLORS.gray8,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of the `textProfileFullName` style has been changed from `colors.gray` to `COLORS.gray8`. As before, ensure that this color change aligns with the overall theme and design guidelines of your application.



<!-- This is an auto-generated comment by OSS CodeRabbit -->